sbnobj/ICARUS/CRT/classes_def.xml

```diff
@@ -8,7 +8,7 @@
 
   <class name="std::map< uint8_t, uint16_t >"/>
   <class name="std::map< unsigned char, std::vector< std::pair<int,float> > > "/>
-
+  <class name="std::pair<unsigned char,std::vector<std::pair<int,float> > > "/>
```

Comment:
Where does the need for this come from?

Reply:
no dictionary for class pair<unsigned char,vector<pair<int,float> > > is available

I was seeing above messages during running the CRTHit reconstruction, adding this line helps. Honestly did not made any effort where it was originating. I will give you more details on it.

Reply:
With the new commit I now see the definition of `CRTHit`, and I think this `std::pair` may be necessary as an internal component of `std::map< unsigned char, std::vector< std::pair<int,float> > >`. Now I am just surprised that it was not caught earlier, since it's pre-existing this pull request, but this is now clarified..

Reply:
Thank you.